code/modules/projectiles/ammunition/_ammunition.dm

```diff
@@ -70,7 +70,7 @@
 			damage_rundown_message += "[return_blows2crit_rating_string(loaded_projectile.damage*pellets)] [loaded_projectile.damage_type] damage"
 
 		if(LAZYLEN(damage_rundown_message))
-			var/start_of_message = "<span class='danger'>Firing it would do "
+			var/start_of_message = "<span class='danger'>Firing it[pellets > 1 ? "s [pellets] pellets" : ""] would do "
```

Comment:
You can just do `\s`

Reply:
The damage is based on all the pellets hitting, I'm trying to allude to that here.

Reply:
I see, nevermind.